docs/using-mycroft-ai/basic-commands.md

```diff
@@ -1,104 +1,328 @@
+##### Set a reminder - *User provides what to say and when*
+> Hey Mycroft, remind me to **turn off the oven** in **5 minutes** <br>
+> Hey Mycroft, add a reminder to **let the dog out** at **3pm**
+##### Set a reminder - *User provides what to say.  Mycroft will ask when.*
+> Hey Mycroft, remind me to **take out the trash**
+##### Set a reminder - *Mycroft will ask what to say and when*
+> Hey Mycroft, set a reminder for me <br>
+> Hey Mycroft, remind me of something
+##### Set a reminder - *User provides just the when.  Mycroft will remind with a beep.*
+> Hey Mycroft, remind me at **5pm**
+##### Snooze the active reminder
+> Hey Mycroft, snooze the reminder <br>
+> Hey Mycroft, remind me later 
+##### Cancel the active reminder
+> Hey Mycroft, okay, stop the reminder
+##### Get reminders for a specific day
+> Hey Mycroft, what are my reminders for today? <br>
+> Hey Mycroft, do I have any reminders tomorrow?
+##### Get next reminder
+> Hey Mycroft, what is my next reminder?
+##### Delete all reminders for a specific day 
+> Hey Mycroft, cancel today's reminders <br>
+> Hey Mycroft, cancel all reminders for tomorrow
+##### Delete all reminders
+> Hey Mycroft, delete all reminders
+###
+### Singing - *Mycroft will speak the lyrics to a random pop music song*
+> Hey Mycroft, sing <br>
+> Hey Mycroft, sing me a song
+###
+### Speak - *Speak a phrase and have Mycroft repeat it*
+> Hey Mycroft, speak **Mary had a little lamb** <BR>
+> Hey Mycroft, say **yada yada yada** <BR>
+> Hey Mycroft, repeat **hey good looking**
+###
+### Spelling - *Mycroft will spell any word that is understood by the Speech-To-Text system.*
+> Hey Mycroft, spell **aardvark** <BR>
+> Hey Mycroft, what is the spelling of **aardvark**? <BR>
+> Hey Mycroft, how do you spell word **aardvark**?
+###
+### Stock - *This Skill has been disabled.*
+###
+### Stop - *Tell Mycroft to STOP doing whatever it is doing.  Plus miscellaneous Device hardware commands.*
+##### Stop - *This command should terminate any command that is in progress*
+> Hey Mycroft, stop <BR>
+> Hey Mycroft, silence <BR>
+> Hey Mycroft, shut up
+##### Reboot the Mycroft Device
+> Hey Mycroft, reboot
+##### Shutdown the Mycroft Device
+> Hey Mycroft, shutdown <br>
+> Hey Mycroft, power down
+##### Activate WIFI 
+> Hey Mycroft, setup wifi <br>
+> Hey Mycroft, configure wifi
+##### SSH Access 
+> Hey Mycroft, enable/disable SSH login <br>
+> Hey Mycroft, allow/deny remote connection
+###
+### Support Helper - *Captures troubleshooting information from the Mycroft Device.*
+> Hey Mycroft, contact support <br>
+> Hey Mycroft, create a support ticket <br>
+> Hey Mycroft, troubleshoot my device <br>
+> Hey Mycroft, create a troubleshoot request <br>
+
+*Captured information is useful for your own debugging and for communicating with Support. The information is emailed to the address registered at home.mycroft.ai.*
```

Comment:
it's uploaded to 0x0.st and the link is emailed to the registered email address

Reply:
Right.